Neos.Flow/Tests/Functional/Security/Authorization/Privilege/Entity/Doctrine/ContentSecurityTest.php

```diff
@@ -81,7 +81,9 @@ protected function setUp(): void
      */
     public function administratorsAreAllowedToSeeHiddenRestrictableEntities()
     {
-        $this->authenticateRoles(['Neos.Flow:Administrator']);
+        $account = $this->authenticateRoles(['Neos.Flow:Administrator']);
+        $account->setAccountIdentifier('Administrator');
+        $account->setAuthenticationProviderName('SomeProvider');
```

Comment:
Why is this required now?

Reply:
With the getAccountIdentifier method that returns a AccountIdentifier object, we create it from the string property `$accountIdentifier` - but it's "null" in this case. Causing the persistence to throw a exception like "argument must be string, null given"
What is wrong here in the test cases, is that we try to persist a model, with required properties (NotEmpty) that has no values. But since the "validation" is using the `get`method to create a value, the before mentioned (null given) exception, is causing the issue